src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -1577,7 +1587,8 @@ export class StudyViewPageStore
     private createMutatedGeneComparisonSession(
         chartMeta: ChartMeta,
         hugoGeneSymbols: string[],
-        statusCallback: (phase: LoadingPhase) => void
+        statusCallback: (phase: LoadingPhase) => void,
+        mutationTypes?: string[]
```

Comment:
what are the types here?  Is there an enum?

Reply:
param removed, I deleted the groupby logic down below, which uses mutationTypes